### PR TITLE
Chore: standardize ZTP references in OCP docs

### PR DESCRIPTION
**Pull request by aireilly**
Standardize references to GitOps ZTP using `{ztp}` and `{ztp-first}` attributes.

Merge to main, CP to enterprise-4.13+

```asciidoc
:ztp-first: GitOps Zero Touch Provisioning (ZTP)
:ztp: GitOps ZTP
```

Preview: 
* Search for "GitOps ZTP" and "GitOps Zero Touch Provisioning (ZTP)" in sections below [Clusters at the network far edge](https://59196--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-clusters-at-scale.html) 

QE review:
Not required.

### CODE REVIEW

**Review by kcarmichael08 — COMMENTED** [review]
Just making sure that these variables have been added (or maybe they were already in) the common-attributes.adoc file. (I work on a different branch of the repo, so I'm not sure)

File: modules/ztp-customizing-the-install-extra-manifests.adoc [hunk 1]
```diff
@@ -4,9 +4,9 @@
 
 :_module-type: PROCEDURE
 [id="ztp-customizing-the-install-extra-manifests_{context}"]
-= Customizing extra installation manifests in the ZTP GitOps pipeline
+= Customizing extra installation manifests in the {ztp} pipeline
 
-You can define a set of extra manifests for inclusion in the installation phase of the zero touch provisioning (ZTP) GitOps pipeline. These manifests are linked to the `SiteConfig` custom resources (CRs) and are applied to the cluster during installation. Including `MachineConfig` CRs at install time makes the installation process more efficient.
+You can define a set of extra manifests for inclusion in the installation phase of the {ztp-first} pipeline. These manifests are linked to the `SiteConfig` custom resources (CRs) and are applied to the cluster during installation. Including `MachineConfig` CRs at install time makes the installation process more efficient.
```
[the remark this answers is not in the document]
Comment (aireilly) [reply modules/ztp-customizing-the-install-extra-manifests.adoc:9]: In this instance, I think that "GitOps ZTP" is OK in the title. The full form would make the title very long.

Per ISG: 
"Avoid using abbreviations in headings and titles unless an abbreviation is well known or the spelled-out form is too long. If you use an abbreviation for the first time in a heading or title, ensure that you include the spelled-out form either in the heading or title or in the text that immediately follows the heading or title."

https://www.ibm.com/docs/en/ibm-style?topic=grammar-abbreviations#headings-and-titles

File: modules/ztp-precaching-troubleshooting.adoc
[the remark this answers is not in the document]
Comment (aireilly) [reply modules/ztp-precaching-troubleshooting.adoc:1]: yes - comments are incorrect.

File: modules/installing-ocp-agent-verify.adoc
[the remark this answers is not in the document]
Comment (aireilly) [reply modules/installing-ocp-agent-verify.adoc:1]: Not sure what happened - removed it.

**Pull request merged**